src/Common/Spectra/PowerSpectrum.jl

```diff
@@ -0,0 +1,127 @@
+using FFTW
+"""
+  PowerSpectrum(u,v,w,L,dim,nor)
+  Calculates the Powerspectrum from the 3D velocity fields u,v,w
+  inputs need to be equi-spaced and it the domain is assumed to be the same
+  size in all direction and having the same number of points in all directions
+  L : size domain
+  dim : number of points
+  nor : normalization factor
```

Comment:
```suggestion
    PowerSpectrum(u, v, w, L, dim, nor)

Calculates the Powerspectrum from the 3D velocity fields `u`, `v`, `w`. Inputs
need to be equi-spaced and the domain is assumed to be the same size and
have the same number of points in all directions.

# Arguments
- L: size domain
- dim: number of points
- nor: normalization factor
```

Reply:
Take a look at [this](https://docs.julialang.org/en/v1/manual/documentation/index.html) in general.